libs/model/src/services/stakeHelper.ts

```diff
@@ -72,7 +72,9 @@ export async function getVotingWeight(
 
     return commonProtocol.calculateVoteWeight(stakeBalance, stake.vote_weight);
   } else if (topic.weighted_voting === TopicWeightedVoting.ERC20) {
-    const { eth_chain_id, private_url, url } = topic.ChainNode!;
+    // if topic chain node is missing, fallback on community chain node
+    const chainNode = topic.ChainNode || community.ChainNode || ({} as any);
```

Comment:
This will still break if `topic.ChainNode` and `community.ChainNode` are null no?

Reply:
If `community.ChainNode` is never null we should do `community.ChainChain!`

Reply:
yep, community *should* never be null in this case